plugins/dell/fu-plugin-dell.c

```diff
@@ -729,12 +729,16 @@ fu_plugin_dell_detect_tpm (FuPlugin *plugin, GError **error)
 	fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_INTERNAL);
 	fu_device_add_flag (dev, FWUPD_DEVICE_FLAG_REQUIRE_AC);
 	fu_device_add_icon (dev, "computer");
-	if (out->flashes_left > 0) {
+	if (!((out->status) & TPM_OWN_MASK) &&
```

Comment:
Wouldn't `if (out->status & TPM_OWN_MASK == 0)` be clearer? I'm not sure whether `TPM_OWN_MASK` means "can own" or "is owned".

Reply:
It means "is owned".  I'll make the logic clearer.